auth-web/src/util/config-helper.ts

```diff
@@ -0,0 +1,38 @@
+import Axios from 'axios'
+
+/**
+ * the configs are used since process.env doesnt play well when we hae only one build config and multiple deployments..so going for this
+ */
+let url = '/static/config/configuration.json'
+export default {
+  fetchConfig () {
+    return Axios
+      .get(url)
+      .then(response => {
+        sessionStorage.setItem('API_CONFIG', JSON.stringify(response.data))
+      }).catch(err => {
+        throw err
+      }
+      )
+  },
+
+  /**
+ * this will run everytime when vue is being loaded..so do the call only when session storage doesnt have the values
+ */
+  saveConfigToSessionStorage () {
+    console.log('sessionStorage.getItem(\'API_CONFIG\')'+sessionStorage.getItem('API_CONFIG'))
```

Comment:
Do we want to leave `console.log` statements in? Maybe we could come up with a convention for including these but having something in place for disabling them.  Otherwise it will be quite a bit of work to go back and remove them all once we move into production.  Maybe something like `DEBUG && console.log`? Not sure, and we should probably discuss as a team before deciding anything.

Reply:
Removed all console logs.Also made eslint strict about console.log.No more console logs :)